halo2-base/src/lib.rs

```diff
@@ -308,6 +383,8 @@ impl<F: ScalarField> Context<F> {
             .collect()
     }
 
+    /// Assigns a witness value and returns the last assigned cell of the `advice` column.
```

Comment:
the last assigned cell is the witness just assigned. So you can say "Assigns a witness value and returns the corresponding assigned cell."